bosh/releases/pre_render_scripts/uaa/uaa/jobs/patch_pre-start.sh

```diff
@@ -36,6 +36,11 @@ patch --verbose "${target}" <<'EOT'
 +        mv /var/lib/ca-certificates/ca-bundle.pem /etc/ssl/certs/"$(basename "${OS_CERTS_FILE}")"
 +      ;;
 +
++      *rhel|centos*)
++        timeout --signal=KILL 180s /usr/bin/update-ca-trust -f -v
++        cp /etc/ssl/certs/ca-bundle.crt ${OS_CERTS_FILE}
```

Comment:
`OS_CERTS_FILE` is [`/etc/ssl/certs/uaa-ca-certificates.crt`](https://github.com/cloudfoundry/uaa-release/blob/v74.25.0/jobs/uaa/templates/bin/pre-start.erb#L125)… I guess this makes sense, then; not sure why the SUSE case looks so weird (but you don't have to touch it).

Reply:
> not sure why the SUSE case looks so weird

I think it was just copying the `"$(basename "${OS_CERTS_FILE}")"` expression from the Ubuntu case without thinking any further. I think it would make sense to simplify the SUSE command as well.